config/settings.yml

```diff
@@ -317,6 +317,15 @@ discourse_ai:
   ai_summarization_discourse_service_api_key:
     default: ""
     secret: true
+  ai_summarization_strategy:
```

Comment:
I think we are missing a way to migrate the old setting here? We don't want to break summaries